docs/sources/helm-charts/mimir-distributed/run-production-environment-with-helm/_index.md

```diff
@@ -263,56 +263,36 @@ For more information, see [Installing Grafana Mimir dashboards and alerts].
 
 The `mimir-distributed` Helm chart makes it easy for you to collect metrics and
 logs from Mimir. It assigns the correct labels for you so that the dashboards
-and alerts simply work. The chart uses the Grafana Agent to ship metrics to
-a Prometheus-compatible server and logs to a Loki or GEL (Grafana Enterprise
-Metrics) server.
+and alerts work.
 
-1. Download the Grafana Agent Operator Custom Resource Definitions (CRDs) from
-   https://github.com/grafana/agent/tree/main/operations/agent-static-operator/crds
-2. Install the CRDs on your cluster:
+{{< admonition type="caution" >}}
+The `mimir-distributed` Helm chart uses Grafana Agent, which is now deprecated. Use Grafana Alloy to ship metrics and logs instead.{{< /admonition >}}
 
-   ```bash
-   kubectl apply -f operations/agent-static-operator/crds/
-   ```
-
-3. Add the following YAML snippet to your
-   values file, to send metamonitoring telemetry from Mimir. Change the URLs and credentials to match your desired
-   destination.
+To send metamonitoring telemetry from Mimir, add the following YAML snippet to your
```

Comment:
Let's verify that this is the correct configuration.

Reply:
Looks correct to me.